src/parser/core/Events.ts

```diff
@@ -851,6 +853,24 @@ export interface SpendResourceEvent extends Event<EventType.SpendResource> {
   __fabricated: true;
 }
 
+export interface CreateEventTarget {
```

Comment:
Is this the same type of Target as all the other Targets in the Event.ts file? Like BeginCastEvent/BaseCastEvent
if so it might be logical to normalize them as well

Reply:
It's slightly different from what I could tell, but will double check